source/docs/03-metadata.xml

```diff
@@ -1160,7 +1160,7 @@
                            <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/header/header-sample063.txt" parse="text"/></egXML>
                         </figure>
                      </p>
-                     <p>Music for a single player should, however, never use the <att>solo</att> attribute.</p>
+                     <p>An ad libitum part, i.e. not essential for the performance of the work, may be marked with the <att>adlib</att> attribute.</p>
```

Comment:
```suggestion
                     <p>An <rend rend="italic">ad libitum</rend> part, i.e. not essential for the performance of the work, may be marked with the <att>adlib</att> attribute.</p>
```

Reply:
NB: Please check that I got the TEI markup right -- @rettinghaus ?

Reply:
@ahankinson `<rend rend="italic">` is MEI code, in TEI it should be `<hi rend="italic">`, I think, but please correct me if I'm wrong!

Reply:
OK! I've made the change to the suggestion.